statistics/update.go

```diff
@@ -376,7 +377,7 @@ func needAnalyzeTable(tbl *Table, limit time.Duration) bool {
 	if time.Since(t) < limit {
 		return false
 	}
-	if AutoAnalyzeRatio >= 0 && float64(tbl.ModifyCount)/float64(tbl.Count) > AutoAnalyzeRatio {
+	if AutoAnalyzeRatio > 0 && float64(tbl.ModifyCount)/float64(tbl.Count) > AutoAnalyzeRatio {
```

Comment:
We make sure the `AutoAnalyzeRatio` is greater than 0, then we can remove the `> 0` check.